magento/magento2
# Introduced granulated Media Gallery ACL resources

### Description (*)


### Related Pull Requests
https://github.com/magento/partners-magento2-infrastructure/pull/35


### Fixed Issues

Fixes magento/adobe-stock-integration#1487: Introduce granulated Media Gallery ACL resources and enforce for old media gallery

### Approval Request

https://jira.corp.magento.com/browse/MC-37384

### Manual testing scenarios (*)
```
** Upload assets
** Edit assets
** Delete assets
** Insert assets into the content
** Create folder
** Delete folder
```

1. Create new admin user
2. Try to apply all mentioned acl resources for this user one by time
3. Verify functionality in old media gallery && new media gallery

### Questions or comments


### Contribution checklist (*)
